SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -1331,6 +1314,47 @@ USpatialNetConnection* USpatialNetDriver::AcceptNewPlayer(const FURL& InUrl, FUn
 	return bOk ? SpatialConnection : nullptr;
 }
 
+USpatialActorChannel* USpatialNetDriver::GetOrCreateSpatialActorChannel(UObject* TargetObject, USpatialNetConnection* InConnection)
```

Comment:
Since you call `GetSpatialOSNetConnection()` when calling `CreateSpatialActorChannel` we don't need to pass it here.
```suggestion
USpatialActorChannel* USpatialNetDriver::GetOrCreateSpatialActorChannel(UObject* TargetObject)
```